src/test/integration/lint_test.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+import * as fs from 'fs';
+import * as os from 'os';
+import {assert} from 'chai';
+import * as path from 'path';
+import {runCommand} from './run-command';
+import {invertPromise} from '../util';
+
+const fixturePath =
+    path.join(__dirname, '../../../src/test/integration/fixtures/');
+
+suite('polymer lint', function() {
+
+  const binPath = path.join(__dirname, '../../../bin/polymer.js');
+
+  this.timeout(2 * 1000);
+
+  test.only('handles a simple correct case', async () => {
```

Comment:
Should this test be focused?

Reply:
Good catch! Thank you thank you thank you!